app/models/manageiq/providers/ovirt4/infra_manager.rb

```diff
@@ -0,0 +1,29 @@
+require 'ovirtsdk4'
+
+class ManageIQ::Providers::Ovirt4::InfraManager < ManageIQ::Providers::InfraManager
```

Comment:
We had that discussion already, and the approach was [presented in the summit](http://www.slideshare.net/ManageIQ/ovirt-juan-hernandez-piotr-kliczewski-manageiq-design-summit-2016) a few weeks ago. However, I have no objection to start with a feature toggle, if you prefer that.